ui/client/package.json

```diff
@@ -6,9 +6,10 @@
   "license": "UNLICENSED",
   "private": true,
   "scripts": {
+    "build": "NODE_ENV=production webpack -p --progress && cp -R assets/icons/license dist",
+    "flow": "flow",
     "start": "PROXY_API_DOMAIN=http://localhost:8081/api webpack-dev-server",
     "start-backend-demo": "PROXY_API_DOMAIN=https://staging.nussknacker.io/api webpack-dev-server",
-    "build": "NODE_ENV=production webpack -p --progress && cp -R assets/icons/license dist",
     "test": "./node_modules/karma/bin/karma start --single-run"
```

Comment:
Does flow run with tests? It will be good to tests flow syntax at test or we can do it after tests.